GitExtUtils/GitCommandConfiguration.cs

```diff
@@ -27,6 +27,7 @@ static GitCommandConfiguration()
             Default.Add(new GitConfigItem("diff.mnemonicprefix", "false"), "diff");
             Default.Add(new GitConfigItem("diff.ignoreSubmodules", "none"), "diff", "status");
             Default.Add(new GitConfigItem("core.safecrlf", "false"), "diff");
+            Default.Add(new GitConfigItem("protocol.file.allow", "always"), "submodule"); // git version 2.38.1 and later disabled file protocol by default
```

Comment:
Why not [`user`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-protocolallow)?

Reply:
https://github.com/gitextensions/gitextensions/pull/10389#discussion_r1020801678

Reply:
user is what is happening by default.  

https://git-scm.com/docs/git-config#Documentation/git-config.txt-protocolallow